nh_clinical/patient.py

```diff
@@ -84,6 +84,16 @@ def _get_name(self, cr, uid, ids, fn, args, context=None):
             result[r['id']] = self._get_fullname(r)
         return result
 
+    def name_get(self, cr, uid, ids, context=None):
+        if isinstance(ids, list):
+            ids = ids[0]
+        names = self.read(cr, uid, ids, [
+            'family_name',
+            'given_name',
+            'middle_names'
+        ], context=context)
+        return [(ids, self._get_fullname(names))]
```

Comment:
It's wrapped in a list due to some code up the inheritance chain that returns the value to the frontend returning the first item and as we need to return the tuple it was always returning the ID and not the full tuple.

I'm not sure why this is but this is what I was talking about on Slack yesterday